src/UI/IdleTimer/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { useIdleTimer } from 'react-idle-timer';
+import { getIsLoggedIn, logout as dappLogout } from 'utils';
+
+const getTimeout = (minutes: number) => 1000 * 60 * minutes;
+const debounce = 500;
+
+interface IdleTimerType {
+  minutes?: number;
+  onLogout?: (props?: any) => void;
+}
+
+export const IdleTimer = ({ minutes = 10, onLogout }: IdleTimerType) => {
+  const isLoggedIn = getIsLoggedIn();
+  const defaultLogout = () => dappLogout();
```

Comment:
I think you can just do  const logout = onLogout ? onLogout : dappLogout and skip this part